client/src/Components/Layout/AdminEventDetails.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import { Grid, Box, Typography, Button, Paper } from '@material-ui/core';
+import { Redirect } from 'react-router-dom';
+import {
+  QueryBuilder as QueryBuilderIcon,
+  Room as RoomIcon,
+  EventNote as EventNoteIcon,
+} from '@material-ui/icons';
+import axios from 'axios';
+import Cookies from 'js-cookie';
+import { withStyles } from '@material-ui/core/styles';
+import { red } from '@material-ui/core/colors';
+import LoaderProgress from './../Common/LoaderProgress';
+import '../../Theme/Css/App.css';
+
+const useStyles = (theme) => ({
+  root: { 'text-align': 'center' },
+  red: {
+    color: red[500],
+  },
+  btnCancel: {
+    borderRadius: 15,
+    width: 230,
+    height: 33,
+    background: '#FBF3F1',

```

Comment:
One thing you can do here (it's totally optional) - I like to make a constants file that exports all of the colours used, and import them from there.

This means that you can define one colour palette to use in all of your components, which stops you from ending up with slightly different colours in different places, and you only need to change them in one place if they're updated